### PR TITLE
Port tests to their respective packages

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR ports package tests to their respective package folders

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: packages/valory/skills/apy_estimation_abci/tests/test_behaviours.py [hunk 1]
```diff
@@ -112,16 +120,37 @@
...
+
+@pytest.fixture(scope="module")
+def ipfs_daemon() -> Iterator[bool]:
+    """Starts an IPFS daemon for the tests."""
+    print("Starting IPFS daemon...")
+    daemon = IPFSDaemon()
+    daemon.start()
+    yield daemon.is_started()
+    print("Tearing down IPFS daemon...")
+    daemon.stop()
```
Comment (Adamantios) [inline packages/valory/skills/apy_estimation_abci/tests/test_behaviours.py:150]: Let's open an issue to add these re-usable fixtures to the test tools.

File: tests/conftest.py [hunk 2]
```diff
@@ -70,7 +70,9 @@
     DEFAULT_ABCI_PORT,
     DEFAULT_TENDERMINT_PORT,
     FlaskTendermintDockerImage,
-    TendermintDockerImage,
+)
+from aea_test_autonomy.fixture_helpers import (  # noqa: F401  # pylint: disable=unused-import
+    tendermint,
```
Comment (Adamantios) [inline tests/conftest.py:75]: Will this have any effect?

https://github.com/valory-xyz/open-autonomy/blob/8dea9b23fd6784891aeb41123cb8a2a6415f2d77/plugins/aea-test-autonomy/aea_test_autonomy/fixture_helpers.py#L61-L73

Comment (angrybayblade) [reply tests/conftest.py:75]: pytest will register this a fixture since this is imported here

File: docs/quick_start.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+
+
+# Quick Start
```
Comment (DavidMinarsch) [inline docs/quick_start.md:3]: @angrybayblade is this a merge error? You didn't want to make any docs changes afaik

File: packages/hashes.csv [hunk 4]
```diff
@@ -3,61 +3,61 @@ open_aea/contracts/scaffold,bafybeiba4pdufnxve5hdkla42lixnxwfkyg7s2bvuy6zqbxjsxt
 open_aea/protocols/scaffold,bafybeihhyjuqvp5zqtikkjrwtikg4fr4tsub43n777huxtpeu77g75poze
 open_aea/protocols/signing,bafybeid6f5ceool4evgaxdegs5pkjoivqjk3mo3ehctalswngb5m5c32wm
```
Comment (DavidMinarsch) [inline packages/hashes.csv:4]: Please sync latest `open_aea/signing` folder from latest `open-aea` release tag.

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**